coalib/collecting/Collectors.py

```diff
@@ -154,7 +154,7 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
                                 bears.
     :param bear_globs:          Globs of bears to collect.
     :param kinds:               List of bear kinds to be collected.
-    :param log_printer:         LogPrinter to handle logging.
+    :param log_printer:         log_printer to handle logging.
```

Comment:
``log_printer to handle logging`` this is about ``from coalib.output.printers.LogPrinter import LogPrinter`` this, so I think it was okay the way it was.